crt_portal/cts_forms/tests/test_views.py

```diff
@@ -876,6 +876,92 @@ def test_complaint_id_filter(self):
         self.assertTrue('2 records' in str(response.content))
 
 
+class CRT_Disposition_Tests(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.superuser = User.objects.create_superuser('superduperuser', 'a@a.com', '')
+        self.report_data = SAMPLE_REPORT_1.copy()
+        self.report_data.update({'status': 'closed'})
+        closed_date = "2022-02-01 18:17:52.74131+00"
+        self.report_data.update({'closed_date': closed_date})
+        retention_schedule = RetentionSchedule.objects.get(retention_years=10)
+        self.report_data.update({'retention_schedule': retention_schedule})
+        expiration_date = "2032-02-01 18:17:52.74131+00"
+        self.report_data.update({'expiration_date': expiration_date})
+        self.report_data.update({'eligible_date': "2032-01-01 18:17:52.74131+00"})
+        self.report = Report.objects.create(**self.report_data)
+        self.report_data_2 = SAMPLE_REPORT_1.copy()
```

Comment:
nit: since these are instance variables, it'd be nice to name them something semantically useful.

Since they're a bunch of reports, you could also put them in a dictionary (or SimpleNamespace, which reads a bit nicer), so the usage would be:

```py
import types
import unittest

class TestSomething(unittest.TestCase):

    def setUp(self):
        self.reports = types.SimpleNamespace(
            expired='expired oh no report here',
            eligible='eligible report here',
        )

    def test_something(self):
        self.assertIn('oh no', self.reports.expired)
        self.assertIn('eligible', self.reports.eligible)
```